container-images/tcib/base/os/tobiko/run_tobiko.sh

```diff
@@ -2,12 +2,34 @@
 
 TOBIKO_DIR=/var/lib/tobiko
 
+if [ ! -z ${USE_EXTERNAL_FILES} ]; then
+    mkdir -p $TOBIKO_DIR/.config/openstack
+    cp $TOBIKO_DIR/external_files/clouds.yaml $HOME/.config/openstack/
+fi
+
 pushd ${TOBIKO_DIR}
 
 export OS_CLOUD=default
 echo "export OS_CLOUD=default" > .bashrc
 
+mkdir -p .downloaded-images
+curl https://cloud-images.ubuntu.com/minimal/releases/jammy/release/ubuntu-22.04-minimal-cloudimg-amd64.img -o .downloaded-images/ubuntu-minimal
```

Comment:
Should not ubuntu version be configurable?

Reply:
Also can't be this be fedora or something?

Reply:
Now it is configurable.
Tobiko uses Ubuntu images as advanced images at this moment:
https://github.com/redhat-openstack/tobiko/blob/master/tobiko/openstack/stacks/_ubuntu.py